src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -3,22 +3,129 @@ title: 'Alerts'
+For example, at Sentry, we send a notification every single time an error happens, and the affected user’s email address ends with `@sentry.io`.
 
-### Actions
+[{% asset notifications/alert_conditions.png alt="Dropdown of alert conditions." %}]({% asset notifications/alert_conditions.png @path %})
 
-Currently there are 3 types of actions you can take:
+### List of Conditions
 
-> [{% asset alert-actions.png %}]({% asset alert-actions.png @path %})
+- An event is seen
+- An issue is first seen
+- An issue changes state from resolved to unresolved
+- An issue changes state from ignored to unresolved
+- An event's tags match {key} {match} {value}
+    - key: any tag
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any key's value
+- An issue is seen more than {value} times in {interval}
+    - value: a positive integer
+    - interval: one minute, one hour, one day, one week, or 30 days
+- An issue is seen by more than {value} users in {interval}
+    - value: a positive integer
+    - interval: one minute, one hour, one day, one week, or 30 days
+- An event's {attribute} value {match} {value}
+    - attribute: message, platform, environment, type, exception.type, exception.value, user.id, user.email, user.username, user.ip_address, http.method, http.url, stacktrace.code, stacktrace.module, or stacktrace.filename
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any attribute's value
+- An event's level is {match} {level}
+    - match: equal to, less than or equal to, greater than or equal to
+    - level: fatal, error, warning, info, debug, or sample
 
-An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
\ No newline at end of file
+## Actions
+
+Actions are located in **Project Settings > Alerts > Rules > "New Alert Rules" or "Edit Rule"**.
+
+[{% asset notifications/new_alert_rule.png alt="Buttons to create a New Alert Rule or to Edit an existing rule." %}]({% asset notifications/new_alert_rule.png @path %})
+
+An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
+
+### Action Types
+
+Currently, there are three types of actions you can take:
+
+- Send a notification (for all legacy integrations)
+    - Per-Project Integrations are considered to be legacy integrations
+- Send a notification via {service}
+    - service is one of mail and any configured per-project integration
+- Send a notification to the {workspace} Slack workspace to {channel} and show tags {tags} in notification
+    - to clarify this relates to the global Slack integration, *not* the per-project legacy integration
```

Comment:
maybe add a comma after "to clarify" or remove "to clarify" - this reads weird as is